### PR TITLE
Table fix

### PR DESCRIPTION
**Pull request by JPorry**

### CODE REVIEW

**Comment by jamesplease** [comment]
This looks good! Thanks @JPorry .

Two things:

I'd update the CHANGELOG in version bump commit, and this is a suuuuper small thing but so far I have been omitting the `v` in the version bump commit msg. reference commit: https://github.com/jamesplease/materialish/commit/6f4a3ac99b2adfd2ee821a9bfa2e66d84911d0b6

File: src/table/table.js [hunk 1]
```diff
@@ -241,21 +241,40 @@ class TableExpandedRowContent extends Component {
...
     this.el.style.display = null;
+
+    if (!this.props.open) {
+      if (initialBb.height) {
+        this.el.style.setProperty(
+          '--mt-table-cellHeight',
+          `${initialBb.height}px`
+        );
+        requestAnimationFrame(() => {
+          this.el.style.setProperty('--mt-table-cellHeight', 0);
```
Comment (jamesplease) [inline src/table/table.js]: Check for the existence of `this.el` here. It could be unmounted.

File: src/table/table.js [hunk 2]
```diff
@@ -241,21 +241,40 @@ class TableExpandedRowContent extends Component {
...
+      this.el.style.setProperty('--mt-table-cellHeight', 0);
+      return;
+    }
+
     requestAnimationFrame(() => {
       this.el.style.setProperty('--mt-table-cellHeight', `${bb.height}px`);
+      this.el.addEventListener(
+        'transitionend',
+        () => {
+          this.el.style.setProperty('--mt-table-cellHeight', 'auto');
```
Comment (jamesplease) [inline src/table/table.js]: I would check for the existence of the `el` here.

File: src/table/table.js [hunk 3]
```diff
@@ -241,21 +241,40 @@ class TableExpandedRowContent extends Component {
...
+          this.el.style.setProperty('--mt-table-cellHeight', 0);
+        });
+        return;
+      }
+      this.el.style.setProperty('--mt-table-cellHeight', 0);
+      return;
+    }
+
     requestAnimationFrame(() => {
       this.el.style.setProperty('--mt-table-cellHeight', `${bb.height}px`);
```
Comment (jamesplease) [inline src/table/table.js]: I'd check for `this.el` before doin' this

File: src/table/table.js (hunk 1 above)
Comment (JPorry) [reply src/table/table.js]: gooooood catch!

**Pull request merged**